### ISSUE TITLE
Batch of variable length sentence

### ISSUE THREAD

**Issue by blablahaha** [airalcorn2/Deep-Semantic-Similarity-Model#9]
Hi,

Thank you so much for making this code! 

For the variable length input, if I understand you correctly, it just can fit one query by one query instead of fitting all data? So I would like to know, whether there exist some *tricks*  to fit all of the  variable length input in one batch?

**Comment by airalcorn2** [comment]
You'll need to use [a Masking layer](https://keras.io/layers/core/#masking).
